scss/_base_code.scss

```diff
@@ -20,10 +21,10 @@
   }
 
   pre {
-    background: $color-x-dark;
-    border: 0;
-    border-radius: .5rem;
-    color: $color-mid-dark;
+    background-color: $color-light;
+    border: 1px solid $color-mid-dark;
+    border-radius: 2px;
```

Comment:
@yaili so, if there is no difference, why would not just stay consistent and use `rem` everywhere?